src/lightning_app/utilities/secrets.py

```diff
@@ -0,0 +1,19 @@
+from typing import Dict, List
+
+from lightning_app.utilities.cloud import _get_project
+from lightning_app.utilities.network import LightningClient
+
+
+def _names_to_ids(secret_names: List[str]) -> Dict[str, str]:
+    """Returns the name/ID pair for each given Secret name."""
+    lightning_client = LightningClient()
+
+    project = _get_project(lightning_client)
+    secrets = lightning_client.secret_service_list_secrets(project.project_id)
+
+    secret_names_to_ids: Dict[str, str] = {}
+    for secret in secrets.secrets:
+        if secret.name in secret_names:
+            secret_names_to_ids[secret.name] = secret.id
```

Comment:
Does that mean you first have to create the secret in the admin? Does that mean I can't launch an app for the first time and immediately set a secret env var?

Reply:
> Does that mean you first have to create the secret in the admin?

Yes, first you create a Secret using the admin UI. Then, you run your app and reference the Secret using `--secret`. This is to allow Secrets to be re-used across multiple different apps (for example, AWS credentials).

> Does that mean I can't launch an app for the first time and immediately set a secret env var, all through cli?

Correct, this isn't possible in this initial implementation. We could definitely add this in the future (along with full CRUD on Secrets using the CLI).